src/only-virtual/OnlyVirtualPage.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default class OnlyVirtualPage extends React.Component {
+    render() {
+        return <div>This site currently in Beta version, only Virtual Accounts are allowed.</div>;
```

Comment:
Nope, it is a showError. Currently alert, later whatever we want it to be.
Instead of rebuilding these components with just a div and a text inside them, dozen times...